docs/README.md

```diff
@@ -1 +1,116 @@
+
+1. Adding the app as a theme dependency in the `manifest.json` file;
+2. Declaring the app's main block in a given theme template or inside another block from the theme.
+
+Remember to add a table with all blocks exported by the app and their descriptions. You can verify an example of it on the [Search Result documentation](https://vtex.io/docs/components/all/vtex.search-result@3.56.1/). 
+
+Next, add the **props table** containing your block's props. 
+
+If the app exports more than one block, create several tables - one for each block. For example:
+
+### `block-1` props
+
+| Prop name    | Type            | Description    | Default value                                                                                                                               |
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+
+
+### `block-2` props
+
+| Prop name    | Type            | Description    | Default value                                                                                                                               |
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+
+Prop types are: 
+
+- `string` 
+- `enum` 
+- `number` 
+- `boolean` 
+- `object` 
+- `array` 
+
+When documenting a prop whose type is `object` or `array` another prop table will be needed. You can create it following the example below:
+
+- `propName` object:
+
+| Prop name    | Type            | Description    | Default value                                                                                                                               |
+| ------------ | --------------- | --------------------------------------------------------------------------------------------------------------------------------------------- | ---------- | 
+| `XXXXX`      | `XXXXXX`       | XXXXXXXX         | `XXXXXX`        |
+
+
+Remember to also use this Configuration section to  **showcase any necessary disclaimer** related to the app and its blocks, such as the different behavior it may display during its configuration. 
+
+## Modus Operandi *(not mandatory)*
+
+There are scenarios in which an app can behave differently in a store, according to how it was added to the catalog, for example. It's crucial to go through these **behavioral changes** in this section, allowing users to fully understand the **practical application** of the app in their store.
+
+If you feel compelled to give further details about the app, such as it's **relationship with the VTEX admin**, don't hesitate to use this section. 
```

Comment:
This is just an instruction to clarify which info should be added to the doc! If no info will be actually added regarding the app behavior, the Modus Operandi section can be removed!